sinonjs/sinon
# Create downloads page (#1218)

#### Purpose (TL;DR) 
Implement task #1218 by:
- copying released package into docs
- creating download page with links,
- add links to releases on release page

#### How to verify
1. Check out this branch
2. `bundle exec jekyll serve`
3. Go to http://127.0.0.1:4000/sinon/releases/download/
4. Go to http://127.0.0.1:4000/sinon/releases/
